learn/app-development/app-security/openid-auth.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Configuring Open ID authentication using SSO"
+id: "openid-auth"
+sidebar_label: "Open ID Authentication"
+---
+In this document, we will be describing about the OpenID authentication using Identity providers.
+
+---
+
+OpenID Auth is a protocol that allows you to log into applications with the help of credentials, provided by an OpenID provider. Authenticated IDs of users let them use multiple applications without creating a separate account and password for each. Well-known OpenID providers are Google, Okta, AuthO and so on.
+
+### Example
+
+Twitter is a well-known social media application that uses OpenID authentication. As soon as you try to login, you are provided with few identity providers like Google and Apple. You can then choose a desired identity provider to log into Twitter.
+
+
+## Why OpenID Authentication
+
+OpenID Authentication comes with many advantages. Important reasons to choose OpenID authentication are:
+
+### Easy maintainance of usernames and passwords
+
+OpenID helps us to reduce efforts in keeping track of multiple usernames and passwords. Developers also find it beneficial as they are not required to design additional logic to store and maintain the usernames and passwords, specific to any application.
```

Comment:
In this sentence you are referring to two different personas.

Application Developers & User of the application. 

When you refer with "us" it is not clear which persona you are talking about.

OpenID helps us to reduce efforts in keeping track of multiple usernames and passwords. Developers also find it beneficial as they are not required to design additional logic to store and maintain the usernames and passwords, specific to any application.

Instead

OpenID stops proliferation of username, passwords enabling reuse of identify from providers such as Google, Apple & Facebook. Developers also benefit from not needing store usernames and passwords.